src/doc/template/tmpl/layout.tmpl

```diff
@@ -161,13 +161,13 @@
             </span>
         </footer>
 
-        <script src="scripts/docstrap.lib.js"></script>
-        <script src="scripts/toc.js"></script>
+        <script src="/britecharts/scripts/docstrap.lib.js"></script>
```

Comment:
github.io serves docs/ directory as http://eventbrite.github.io/britecharts/ so it would not be a problem.